test/integration/cms/api/files_test.rb

```diff
@@ -116,7 +116,7 @@ def attachment_for_upload
         assert_response :success
 
         doc = Nokogiri::XML(@response.body)
-        id = doc.xpath('//id').text
+        id = doc.xpath('/file/id').text
```

Comment:
Out of curiosity: Why did the paths in those tests changed?

Reply:
Done

Reply:
Because before the change, there was only one `<id>` in the response, but now there are more than one, e.g. `<file><section><id>`, so we need to specify which one